packages/launchpad-navigator/README.md

```diff
@@ -0,0 +1,34 @@
+# Launchpad Navigator
+
+This package provides the components for attaching the Navigator of the 
+Launchpads to the master bar.
```

Comment:
I don't believe this is strictly true. This package provides the components for showing and interacting with one or more Launchpad task lists for a given site.